examples/smartcore-ml/README.md

```diff
@@ -0,0 +1,119 @@
+
+## Model Types
+
+It is important to specificy the generic types for each model so that they can be succesfully deserialized.  The input data for a trained model is typically formatted as a DenseMatrix, so be sure to add the following import: 
+`use smartcore::linalg::basic::matrix::DenseMatrix;`
+when using KNN, be sure to add the following import:
+`use smartcore::metrics::distance::euclidian::Euclidian;`
+A list of the supported models with definitions for their respective types can be found below.  Note that {float} is a placeholder for either f32 or f64:
+
+```
+let model: DecisionTreeClassifier<{float}, u32, DenseMatrix<{float}>, Vec<u32>> = trained_model;
+```
+
+```
+let model: RandomForestClassifier<{float}, u8, DenseMatrix<{float}>, Vec<u8>> = trained_model;
+```
+
+```
+let model: GaussianNB<{float}, _, DenseMatrix<{float}>, _> = trained_model;
+```
+
+```
+let model: LogisticRegression<{flaot}, u32, DenseMatrix<{float}>, Vec<u32>> = trained_model;
+```
+
+```
+let model: LinearRegression< {float}, {float}, DenseMatrix<{float}>, Vec<{float}>> = trained_model;
+```
+
+```
+let model: KNNClassifier< {float}, u32, DenseMatrix<{float}>, Vec<u32>, Euclidian<{float}>> = trained_model;
+```
+
+```
+let model: KMeans<{float}, u8, DenseMatrix<{float}>, Vec<u8>> = trained_model;
+```
+
+```
+let model: PCA<{float}, DenseMatrix<{float}>> = trained_model;
+```
+
+NOTE:  For Support Vector Machine classifiers and regression models, you MUST use the following smartcore fork and import into you cargo.toml file:
+
+smartcore = { git = "https://github.com/Roee-87/smartcore.git", features = ["serde"]}
+
+Deserialization of SVC and SVR does not includes that SVCParametersmake field.  It must be added back into the model struct manually after deserialization.  The SVC and SVR fork have ammended the visbility of the model struct, making the parameters field public and thereby allowing the parameters field to be directly inserted into the model struct after dserialization.  You must insert the same parameters that were used when training the model.
+```
+let mut model: SVC<{float}, i32, DenseMatrix<{float}>, Vec<i32>> = deserialized_svc_trained;
+let params = &SVCParameters::default().with_c(200.0).with_kernel(Kernels::linear());
+model.parameters = Some(params);
+```
+
+```
+let mut model: SVR< '_, {float}, DenseMatrix<{float}>, Vec<{float}>> = deserialized_model;
+//make sure to use the same SVRParameters used to train the model
+let params = &SVRParameters::default().with_eps(2.0).with_c(10.0).with_kernel(Kernels::linear());
+model.parameters = params;
+```
+
+SVM models support four kernels:  Gaussian (rbf), Linear, Polynomial, and Sigmoid.  Kernels are selected within the .with_kernel() function in the SVCParameters or SVRParameters.
+Kernel::linear()
+Kernel::rbf()
+Kernel::polynomial()
+Kernel::sigmoid()
```

Comment:
I'm not sure what you're intending with these lines, but the status quo isn't it.

Reply:
Fixed